### PR TITLE
Feature/upload list cover

### PR DESCRIPTION
**Pull request by marouane-erouhi**
### What this PR does (required):
- a list cover can now be uploaded on list creation

### Screenshots / Videos (required):
https://user-images.githubusercontent.com/13183518/122929354-daf43980-d338-11eb-9bf4-c0ea7f0db445.mp4

### Any information needed to test this feature (required):
- Login
- Click create a new list button
- fill out the info in the modal
- submit

### CODE REVIEW

File: client/src/helpers/APICalls/imageUpload.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const uploadImage = async (images: any): Promise<any> => {
+  const fd = new FormData();
+
+  for (const image of images) {
+    fd.append('images', image, `${image.name}`);
+  }
+
+  const fetchOptions: FetchOptions = {
+    method: 'POST',
+    // headers: { 'Content-Type': 'multipart/form-data; boundary=AaB03x' },
```
Comment (moffatethan) [inline client/src/helpers/APICalls/imageUpload.ts:22]: I would add headers in here

**Review by moffatethan — COMMENTED** [review]
Left you a comment to address, also in the future please base your branches properly so we don't have to see code that has already been reviewed. [See GitHub Docs](https://github.com/hatchways/team-athens/pull/74/files#)

**Please address merge conflicts as well**

File: client/src/helpers/APICalls/imageUpload.ts (hunk 1 above)
Comment (marouane-erouhi) [reply client/src/helpers/APICalls/imageUpload.ts:22]: commenting that out was the only way I could get it working, any ideas why?

Comment (moffatethan) [reply client/src/helpers/APICalls/imageUpload.ts:22]: Could you provide some further context into what is happening when that header is set?

Comment (marouane-erouhi) [reply client/src/helpers/APICalls/imageUpload.ts:22]: the image doesn't upload, and on the server-side the body contains nothing. other than that the endpoint returns a Success status.

**Pull request merged**